### ISSUE TITLE
determining which nodeshift to use

### ISSUE THREAD

**Issue by lholmquist** [bucharest-gold/nodeshift#99]
This idea comes from the ember-cli.  like nodeshift, ember-cli is usually installed globally

To use nodeshift, you usually install globally, `npm install -g nodeshift`  so then to use you could use `nodeshift --version`  and this would give you the nodeshift version installed globally.

the same thing happens with ember,  but if you are in an ember project(created with the ember-cli) and you run the `ember` command, you get the version that is installed within that project

i feel like nodeshift should behave the same way.  one way to do that now, is the use a npm script, like `npm run openshift` where the openshift script is `nodeshift [args]`.  since npm uses your locally installed modules, it will use the version of nodeshift installed from your package.json

To summarize, assuming you have nodeshift installed globally, if you run `nodeshift` outside of a project that has nodeshift installed, you use the globally installed version

but if you run `nodeshift` inside a project that has it installed, you will get that locally installed version
